doc/classes/StringName.xml

```diff
@@ -954,18 +954,23 @@
 			<return type="String" />
 			<param index="0" name="right" type="Variant" />
 			<description>
+				Formats the [StringName], replacing the placeholders with one or more parameters, returning a [String]. To pass multiple parameters, [param right] needs to be an [Array].
+				In C#, there is no direct equivalent to this operator. Use the [method format] method, instead.
```

Comment:
I'd prefer if we recommended the $ string interpolation for C#, can we link to https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated?